elixir/apps/web/lib/web/live/sites/show.ex

```diff
@@ -59,71 +59,85 @@ defmodule Web.Sites.Show do
     </.section>
 
     <.section>
-      <:title>
-        Resources
-      </:title>
+      <:title>Gateways</:title>
       <:action>
-        <.add_button navigate={~p"/#{@account}/resources/new?site_id=#{@group}"}>
-          Create
+        <.add_button navigate={~p"/#{@account}/sites/#{@group}/new_token"}>
+          Deploy
         </.add_button>
       </:action>
       <:content>
         <div class="relative overflow-x-auto">
-          <.table
-            id="resources"
-            rows={Enum.reject(@group.connections, &is_nil(&1.resource))}
-            row_item={& &1.resource}
-          >
-            <:col :let={resource} label="NAME">
+          <.table id="gateways" rows={@group.gateways}>
+            <:col :let={gateway} label="INSTANCE">
               <.link
-                navigate={~p"/#{@account}/resources/#{resource}?site_id=#{@group}"}
+                navigate={~p"/#{@account}/gateways/#{gateway.id}"}
                 class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
               >
-                <%= resource.name %>
+                <%= gateway.name_suffix %>
               </.link>
             </:col>
-            <:col :let={resource} label="ADDRESS">
-              <%= resource.address %>
+            <:col :let={gateway} label="REMOTE IP">
+              <code>
+                <%= gateway.last_seen_remote_ip %>
+              </code>
+            </:col>
+            <:col :let={gateway} label="TOKEN CREATED AT">
+              <.created_by account={@account} schema={gateway.token} />
+            </:col>
+            <:col :let={gateway} label="STATUS">
+              <.connection_status schema={gateway} />
             </:col>
             <:empty>
-              <div class="text-center text-slate-500 p-4">No resources to display</div>
+              <div class="flex justify-center text-center text-slate-500 p-4">
+                <div class="w-auto">
+                  <div class="pb-4">
+                    No gateway instances to display.
+                  </div>
+                  <div class="pb-4">
+                    TODO: Gateways should be deployed to your infrastructure,
+                    all gateways within a site should be able to access all site resources,
+                    connections within a site will be load balanced if there are multiple gateways deployed.
+                  </div>
```

Comment:
How would you feel about holding off on this message until we have exactly what we want to say here?

Reply:
Was thinking of using some GPT magic to fill it before this PR is merged :).